### DISCUSSION TITLE
About the Readme

### DISCUSSION THREAD

**Discussion by Korozion** [luskaner/ageLANServer#37] [Q&A] [answered]
Hi,
First, thank you for the effort you put in this project !

In your project Readme you write : 

> You will still need a way to bypass the online-only restriction that is imposed by the game to being connected to the internet and Steam or Xbox Live, depending on the version, to fully play offline.

Is this a hypothetical sentence, or did you ever managed to launch a LAN game without internet connection ? If so, I would be very curious to know how you tricked steam into believing it is connected to the internet ?

**Comment by luskaner** [comment, marked as answer]
Hi:

Glad you like it :)

Yeah I managed to launch it with a PC with no internet at all, I used a recent fork of the famous Goldberg Steam Emulator (https://github.com/Detanup01/gbe_fork) and modified a bit the code so the game believes is always online =) 

PS: For legal reasons it is not provided here on GitHub ;)

**Reply by luskaner** [reply]
Instructions on how to download and configure the emulator is now on https://github.com/luskaner/ageLANServerLauncherCompanion plus the DLL needed.

**Reply by Korozion** [reply]
Works like a charm !
